### DISCUSSION TITLE
How to use the "next" field

### DISCUSSION THREAD

**Discussion by KikoTs** [userbradley/BeReal#20] [Q&A] [answered]
In example I request https://mobile.bereal.com/api/relationships/friends
and i get a JSON containing 3 main fields

```json
{
    "next": "QQAspebQYVc538hQtJbHgG73iTJ3", 
    "total": 0,
    "data": [...]
}
```

How do I use the next field to get more results about my friends as it's not showing the full array?

**Comment by KikoTs** [comment, marked as answer]
Okay did some tries and errors and found out that the request should be like this
https://mobile.bereal.com/api/relationships/friends?page=QQAspebQYVc538hQtJbHgG73iTJ3

wierd they used ?page instead of ?next
Anyways you might want to include this into the docs

**Reply by userbradley** [reply]
Yeah that's a common spec with API designs, as these are technically pages to be served.

**Comment by Lukas1h** [comment]
Good find!

**Comment by KikoTs** [comment]
I will close the issue, thanks for noticing it🙌
